antenna-workflow-steps/processors/validators/antenna-drools-checker/src/test/java/org/eclipse/sw360/antenna/drools/DroolsEngineTest.java

```diff
@@ -25,12 +24,7 @@
 import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
See above, interesting, sometimes you swap this way, sometimes the other way

Reply:
This is standard IntelliJ formatting and we use it everywhere: Once there are a number of imports from the same package (I guess four?), it'll use `*` importing. 

We either need a documented codestyle, or I prefer using the default of IntelliJ that we all seem to have configured (those import changes here are just in classes were no autoformatting has been done).

Reply:
:-(
Fighting the tool does not make sense. As long as we do not reconfigure that manually, it is ok